www/index.js

```diff
@@ -227,8 +229,10 @@ HotPush.prototype._loadLocalVersion = function(callback) {
 * Callback for async call to version files
 */
 HotPush.prototype._verifyVersions = function() {
```

Comment:
what would be awesome is to add an argument to choose whether look for the timestamp or the version number and expose it in the options